src/components/carousel/carousel-items.jsx

```diff
@@ -3,6 +3,7 @@ import Box from "@mui/material/Box";
 import EastIcon from "@mui/icons-material/East";
 import { ProductImage } from "../product-image";
 import { CategoryItems } from "./category-titles";
+import Link from "@mui/material/Link";
```

Comment:
is there a reason to use Link from mui and not the router link ?

Reply:
I've read this:
Integration with Router Library: If you're using a router library like React Router, it's generally recommended to use that library's Link component. This ensures proper integration with the router's navigation system and allows for features like client-side routing without full page reloads.
so I will change it to router link